File: MODUL 2/Python/PRAK205-2310817220011-ReginaSilvaMaharatini.py
import math

print("Input")
angka = input().split()
if len(angka) == 1:
    A = float(angka[0])
    B = float(input())
else:
    A, B = map(float, angka)
    
C = math.sqrt(B**2-A**2)

tinggi = A
alas = C
keliling = A+B+C
luas = 0.5*(C*A)

print("Output")
print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")

File: MODUL 2/C/PRAK205-2310817220011-ReginaSilvaMaharatini.c
#include <stdio.h>
#include <math.h>

int main()
{
    float A, B;

    printf("Input\n");
    scanf("%f %f", &A, &B);

    float C = sqrt(B*B-A*A);
    float tinggi = A;
    float alas = C;
    float keliling = A+B+C;
    float luas = 0.5*(C*A);

    printf("\nOutput\n");
    printf("Alas = %.0f cm\n", alas);
    printf("Tinggi = %.0f cm\n", tinggi);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm^2\n", luas);

    return 0;
}